Musical Collection/Musical Collection Console App/Interfaces/Classes Interfaces/IListener.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Musical_Collection_Console_App.Classes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Interfaces.Classes_Interfaces
+{
+    public interface IListener : IEntity
+    {
+        List<string> FavouriteGenres { get; set; }
```

Comment:
Its preferable to abstract collections as well 
IEnumarble<string> FavouriteGenres is fine